README.md

```diff
@@ -96,6 +100,13 @@ positionals   // ["-"]
 remainder     // ["--baz"]
 argv          // [ { index: 0, type: 'argument', value: { name: "foo", value: "bar" } } ... ]
 ```
+🤔 I'm confused with the returned value of this example here since it seems to
+conflict a bit with the statement above:
+
+> The `type` `"value"` will only ever be defined -- in place of `"positional"` -- when `positionalValues=true`
+
+What "type value" is this referring to? is it `argv`'s `value` property? Or
+maybe it's a top level property that will replace `positional`?
```

Comment:
Sorry, I think this example needs to be updated. The first parsed element is a positional `-` without a value (and that's not shown here). I'll update this.